From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: lib/test_linear_ranges: add missing MODULE_DESCRIPTION() macro
Date: Fri, 31 May 2024 16:45:16 -0700

make allmodconfig && make W=1 C=1 reports:
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/test_linear_ranges.o

Add the missing invocation of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240531-md-lib-test_linear_ranges-v1-1-053a1aad37c6@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Reviewed-by: Matti Vaittinen <mazziesaccount@gmail.com>
Cc: Mark Brown <broonie@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/test_linear_ranges.c |    1 +
 1 file changed, 1 insertion(+)

--- a/lib/test_linear_ranges.c~lib-test_linear_ranges-add-missing-module_description-macro
+++ a/lib/test_linear_ranges.c
@@ -216,4 +216,5 @@ static struct kunit_suite range_test_mod
 
 kunit_test_suites(&range_test_module);
 
+MODULE_DESCRIPTION("KUnit test for the linear_ranges helper");
 MODULE_LICENSE("GPL");
_
